### PR TITLE
Create de.lua

### PR DESCRIPTION
**Pull request by ghost**
Deutsche Sprachfiles

**Describe Pull request**
First, make sure you've read and are following the contribution guidelines and style guide and your code reflects that.
Write up a clear and concise description of what your pull request adds or fixes and if it's an added feature explain why you think it should be included in the core.

If your PR is to fix an issue mention that issue here

**Questions (please complete the following information):**
- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? [yes/no] (Be honest)
- Does your code fit the style guidelines? [yes/no]
- Does your PR fit the contribution guidelines? [yes/no]

### CODE REVIEW

**Review by divide29 — CHANGES_REQUESTED** [review]
Add to error:
`inventory_full = 'Inventar zu voll, um alle möglichen Gegenstände zu bekommen. Stelle sicher, dass dein Inventar nächstes mal leer ist. Verlorene Gegenstände: %{value}'`

File: locales/de.lua [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        subject = 'Schmelzende Gegenstände',
+        message = 'Wir haben Ihre Gegenstände fertig geschmolzen. Du kannst sie jederzeit abholen kommen.',
+        open_pawn = 'Öffnen Sie das Pfandhaus',
+        sell = 'Artikel verkaufen',
+        sell_pawn = 'Gegenstände an das Pfandleihhaus verkaufen',
+        melt = 'Gegenstände schmelzen',
+        melt_pawn = 'Öffnen Sie die Schmelzerei',
+        melt_pickup = 'Geschmolzene Gegenstände abholen',
+        pawn_closed = 'Das Pfandhaus ist geschlossen. Komm zurück zwischen %{value}:00 AM - %{value2}:00 PM',
+        sell_items = "Verkaufspreis $%{value}',
```
Comment (Sharkiiie) [inline locales/de.lua:22]: Has an opening double quote and single closing quote.
Please use single quote for both.

**Review by AkinoKitsu — CHANGES_REQUESTED** [review]
Resolve syntax error, add missing error for `inventory_full`

**Comment by Sharkiiie** [comment]
Gonna bump this, if not resolved the PR will be closed.

**Pull request closed without merging**